actions/retest/main.go

```diff
@@ -143,7 +143,7 @@ func main() {
 				}
 
 				statusList := filterStatusList(rs)
-
+				found := false
```

Comment:
name variable as failedTestFound for better readability?

Reply:
done